src/plugins/intel_gna/legacy/src/convert_function_to_cnn_network.cpp

```diff
@@ -52,6 +52,8 @@
 #include "transformations/rt_info/fused_names_attribute.hpp"
 #include "transformations/rt_info/primitives_priority_attribute.hpp"
 #include "transformations/utils/utils.hpp"
+#include "../../src/ops/gna_convolution.hpp"
```

Comment:
If required path is not part of include path I suggest to add it in CMakeLists.txt plugin to avoid using `..` which may cause problem with moving files and so on.